puppet/manifests/overcloud_compute.pp

```diff
@@ -259,6 +259,36 @@
         require => File['/etc/astute.yaml'],
         path => '/usr/local/bin:/usr/bin:/sbin:/bin:/usr/local/sbin:/usr/sbin',
       }
+    } elsif hiera('fdio', false) {
+      $odl_username  = hiera('opendaylight_username')
+      $odl_password  = hiera('opendaylight_password')
+      $ctrlplane_interface = hiera('nic1')
+      if ! $ctrlplane_interface { fail("Cannot map logical interface NIC1 to physical interface") }
+      $vpp_ip = inline_template("<%= scope.lookupvar('::ipaddress_${ctrlplane_interface}') -%>")
+      $fdio_data_template='{"node" : [
+{"node-id":"<%= @hostname %>",
```

Comment:
@wojd hostname resolves to shorthand hostname of the server.  I need to look into this and make it fqdn